population/population_types.go

```diff
@@ -43,6 +45,12 @@ func (c *Client) GetPopulationTypes(ctx context.Context, input GetPopulationType
 	}
 
 	urlPath := "population-types"
+	if input.Limit > 0 {
```

Comment:
don't think we need these tests, 'limit' and 'offset' will be initialised to 0 when the structure is passed, and currently if 0 is passed as limit code defaults it to '20' So simply adding what ever limit and offset that is passed in pagination should be ok